### ISSUE TITLE
import/export doesn't report errors in reexports

### ISSUE THREAD

**Issue by honzahruby** [9romise/eslint-import-resolver-oxc#72]
No error is reported for multiple exports in barrel files when using `export *`.

**Works**
```ts
export { ThingThatExistsInGenerated } from './generated';

export const ThingThatExistsInGenerated = {}; // is correctly reported by import/export
```

**Doesn't work**
```ts
export * from './generated';

export const ThingThatExistsInGenerated = {}; // should be reported by import/export
```

**Comment by honzahruby** [comment]
My bad, I removed the config for `import/parsers` 🤦
